pkg/build/build_test.go

```diff
@@ -209,8 +208,8 @@ func TestConfiguration_Load(t *testing.T) {
 						"GOPATH":     "/var/cache/melange/go",
 					},
 					Accounts: apko_types.ImageAccounts{
-						Users:  []apko_types.User{{UserName: "build", UID: 1000, GID: apko_types.GID(&gid1000)}},
-						Groups: []apko_types.Group{{GroupName: "build", GID: 1000, Members: []string{"build"}}},
+						Users:  []apko_types.User{{UserName: buildUser, UID: 1000, GID: apko_types.GID(&gid1000)}},
+						Groups: []apko_types.Group{{GroupName: buildUser, GID: 1000, Members: []string{buildUser}}},
```

Comment:
Again, limited understanding of golang declarations, but doesn't `buildUser` need to be defined first?

(But then, how does go test pass?)

Reply:
`buildUser` is accessible from anywhere in the `build` package 👍🏻

Reply:
Nevermind, got where it's coming from.